pkg/datastore/test/watch.go

```diff
@@ -74,7 +74,7 @@ func WatchTest(t *testing.T, tester DatastoreTester) {
 
 			opts := datastore.WatchOptions{
 				Content:                 datastore.WatchRelationships,
-				WatchBufferLength:       128,
+				WatchBufferLength:       50,
```

Comment:
why the change?

Reply:
See the commit message: it was required to fix the test

Reply:
why would a smaller buffer be needed to fix a test?